docs/docs/1.0.6/documentation/health.md

```diff
@@ -0,0 +1,160 @@
+# Health checks
+
+Health is an extremely important part of the microservice architecture for various reasons. Load balancers rely on the 
+health status of the application while picking endpoint from load balancer set. Users want the application state to be 
+observable through the GUI or CLI. Orchestrators like Kubernetes also want to know the application status to manage 
+lifecycle of the application. 
+
+Kuma supports several aspects of the health checking. There are two policies which allows configuring active and passive:
+[Health Check](../policies/health-check.md) and [Circuit Breaker](../policies/circuit-breaker.md).
+
+Kuma is able to track the status of the Envoy proxy. If grpc stream with Envoy is disconnected then Kuma considers this 
+proxy as offline. 
+
+Also, every `inbound` in the Dataplane model has `health` section:
+
+```yaml
+type: Dataplane
+mesh: default
+name: web-01
+networking:
+  address: 127.0.0.1
+  inbound:
+    - port: 11011
+      servicePort: 11012
+      health:
+        ready: true
+      tags:
+        kuma.io/service: backend
+        kuma.io/protocol: http
+```
+
+This `health.ready` status is intended to show the status of the application itself. It is set differently depending on 
```

Comment:
nit: you are talking about application, but in YAML there is service, let's keep one terminology for it. Scan the whole doc for it